src/project_manager/ros_utils.cpp

```diff
@@ -706,6 +761,28 @@ Utils::FileName ROSUtils::getCatkinToolsProfilesYamlFile(const Utils::FileName &
     return profiles;
 }
 
+bool ROSUtils::setCatkinToolsProfilesYamlFile(const Utils::FileName &workspaceDir,
```

Comment:
I just noticed that you created some additional functions and changed a few so not sure if this the above comment is valid.